lib/commons/color/get-background-color.js

```diff
@@ -53,17 +53,17 @@ function getBgColor(elm, elmStyle) {
  * @param {Element} bgNode
  * @return {Boolean}
  */
-function contentOverlapping(targetElement, bgNode) {
+function contentOverlapping(targetElement, fgNode) {
 	// get content box of target element
-	// check to see if the current bgNode is overlapping
-	var targetRect = targetElement.getClientRects()[0];
```

Comment:
The `fgNode` in this case would be the foreground "whatever". The function receives two element parameters - one that's the "background" and one that's potentially overlapping (generally a child element). The only way you'd have overlapping would be with absolute positioning, but you'd still need a parent/child combo, right? Are there any scenarios I'm not covering here?